### ISSUE TITLE
Two-Factor Auth support

### ISSUE THREAD

**Issue by PH4NTOMiki** [cgrossde/Pullover#123]
I had to turn off the 2F Auth to be able to log into your program, and I'm using Windows NT 6.3(Win 8.1 64bit x64)

**Comment by niXta1** [comment]
The only problem I have with the program. 
Disable FA, login, enable FA. :(

**Comment by oG14108149527Exl0Ftr7sY** [comment]
Would be great with  Two-Factor Authentication

**Comment by PH4NTOMiki** [comment]
Yeah, I agree otherwise it's a really great piece of software but that definitely an issue, and having to disable,login,enable 2FA every time I want to re log again isn't really a solution

**Comment by iAmRenzo** [comment]
After 10x trying to log in I remembered this. This issue is really old!

**Comment by b0red** [comment]
So. One year later and this is still not supported? 2 bad, I love this app, but I'd like to use 2fa or somekind of onetime password like you can do with google apps.

**Comment by iAmRenzo** [comment]
Great! I somehow was logged out again on Pushover. And I have to repeat this again. With all the dates I see on the files, I don't think @cgrossde is working on this :(

**Comment by cgrossde** [comment]
Guys, this is open source, feel free to fork and fix it yourself ;) I don't maintain this project anymore.
